app/models/court_application_summary.rb

```diff
@@ -21,6 +21,7 @@ def to_builder
       application_case_summary.applicationTitle court_application.court_application_type.type
       application_case_summary.applicationType court_application.court_application_type.code
       application_case_summary.receivedDate court_application.applicationReceivedDate.to_date
+      application_case_summary.applicationResult court_application.result_code
```

Comment:
Note we still don't 100% know that this is going to be the name of this field in the HMCTS payload, but we need _something_ to develop against, even if we change it later.